actions/createProposal.ts

```diff
@@ -67,6 +68,8 @@ export const createProposal = async (
   proposalIndex: number,
   instructionsData: InstructionDataWithHoldUpTime[],
   isDraft: boolean,
+  isMulti: boolean,
```

Comment:
technically this argument is redundant; we can just look at the number of `options` to see if there's more than one